applications/PolkaBot_Lovable_for_Polkadot.md

```diff
@@ -0,0 +1,189 @@
+**USP:** AI App generators weren’t possible 6 months ago, and enable people to try more project types to see what they can scale.
+
+## 👥 Team
+
+- **Team Name:** PolkaShip
+- **Contact Name:** Jerome Minney
+- **Contact Email:** jeromeminney@gmail.com
+- **Website:** http://github.com/jerrysam
+
+### Team members
+
+ - Jerome Minney (lead developer)
+ - Mofiyin Onanuga (partnerships and growth)
+ - Tom Bizzell (advisor)
+
+#### LinkedIn Profiles (if available)
+
+ - https://www.linkedin.com/in/jeromeminney
+ - https://www.linkedin.com/in/mofiyin-onanuga-ba4020188/
+ - https://www.linkedin.com/in/tom-bizzell
+
+### Team Code Repos
+
+- App builder (Hackathon Progress): https://github.com/jerrysam/lovable-for-polkadot
+- Freight app (started with an App builder): https://github.com/TomBizzell/freight-futures
+
+Please also provide the GitHub accounts of all team members:
+
+ - http://github.com/jerrysam
+
+### Team's experience
+
+ - Worked for DeepMind
+ - Worked for Faculty AI (Europe's largest Applied AI consultancy and OpenAI's first implementation partner)
+ - Worked for Unkillable (helped build 8 different growth teams for ~Series A stage startups)
+ - Founded a company, raised 6 figures, built a product, sold stake as part of a technology acquisition strategy towards Visa.
+ - ex VC
+ - 6 figure startup Founder
+ - Scaled to 100 person Team
+ - Digital & Marketing Strategy lead for 7 figure tech startups across SA,UK,US + AUS
+ - + more
+
+## 📊 Development Status
+
+ - Hackathon MVP proved the process of creating Polkadot Dapps is possible.
+ - Current focus is improving reliability of all components, including IDE, LLM prompt chains, and evals.
+ - Next steps include supporting more common Polkadot Dapps and integration work outlined in the Project Details section.
+
+## 📅 Development Roadmap
+
+**Milestone 1:** PolkaBot IDE / Frontend
+ - Create (and understand) the frontend components and UI required to build dapps in the browser.
+ - Grant team/User can open a live app and prompt for code, edit the code editor, prompt for bash commands, and see the rendered website (output).
+ - KPI: Frontend shown to 3 Polkadot developers and feedback gathered for usability
+
+| item | days | rate |
+|------|------|------|
+| Chat UI | 3 | $250 |
+| Website renderer | 2 | $250 |
+| In-browser filestorage | 2 | $250 |
+| Runtime environment | 3 | $250 |
+| Build/compilation tools | 2 | $250 |
+| Integration of the above | 2 | $250 |
+| Usability research and tweaks | 2 | $250 |
+| **Total** | 16 | $4000 |
+
+
+**Milestone 2:** Polkadot app generation supported
+ - Prompts can now generate a working Polkadot app with high reliability, targeting Polkadot Hub
+ - How to demo: Users can generate apps and see that the code interfaces with polkadot hub and assets can be managed on Polkadot Hub.
+ - Grant team/User can enter a prompt and receive a rendered app, with the ability to refine via follow-up prompts. Works well for one app type.
```

Comment:
Perhaps including starter prompts anyone can use to get an app built and running. This would be nice to include in your deliverables.